src/app/test/request.types.ts

```diff
@@ -4,6 +4,8 @@ export const RunTestsRequestSchema = z.object({
   host: z.string().url(),
   ingressProxyPath: z.string(),
   cdnProxyPath: z.string(),
+  trafficName: z.string(),
+  integrationVersion: z.string(),
```

Comment:
Let's rename the `trafficName` to `integrationName`, I feel like it makes more sense here. WDYT?

Reply:
Integration name can be confused with values like "akamai", "akamai-edgeworker" "akamai-property-rules" or even with repository name. Traffic name means the value we get from traffic monitoring. We can extend the name but I strongly advise using the word traffic in the variable name

Reply:
Okay, let's leave it as it is